open-bot.yaml

```diff
@@ -517,6 +517,53 @@ rules:
         If you think this is still a valid issue, please file a new issue with additional information.
 
 
+# Add action actions box to each pull request
+- filters:
+    pull_request: true
+    open: true
+    not:
+      comment:
+        matching: admin-actions
+        author: webpack-bot
+  actions:
+    comment:
+      identifier: admin-actions
+      message: |-
+        *For maintainers only:*
+
+        * [x] <!-- document --> This need to be documented (issue in webpack/webpack.js.org will be filed when merged)
```

Comment:
Does it create a link?

Reply:
this is a checkbox that if ticked will tell the bot to go and file an issue after pull request is merged. no link is generated or needed here imho